pkgs/by-name/sp/space-cadet-pinball/package.nix

```
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchzip,
+  cmake,
+  SDL2,
+  SDL2_mixer,
+  unrar-wrapper,
+}:
+
+let
+  assets =
+    (fetchzip {
+      url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
+      sha256 = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
+      stripRoot = false;
+    }).overrideAttrs
+      (old: {
+        nativeBuildInputs = old.nativeBuildInputs ++ [ unrar-wrapper ];
+      });
+  darwinApp = "$out/Applications/SpaceCadetPinball.app/Contents";
+  assetsDest =
+    if stdenv.hostPlatform.isDarwin then darwinApp + "/Resources" else "$out/share/SpaceCadetPinball";
+in
+stdenv.mkDerivation rec {
+  pname = "SpaceCadetPinball";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "k4zmu2a";
+    repo = pname;
```

```suggestion
    repo = "SpaceCadetPinball";
```
Avoid interpolating `pname`.